src/api/index.ts

```diff
@@ -171,7 +172,11 @@ router.get(
 				return;
 			}
 
-			res.json(subscription.repoSyncState);
+			if(await booleanFlag(BooleanFlags.REPO_SYNC_STATE_AS_SOURCE, false)) {
```

Comment:
We should pass the `jiraHost` into the feature flag evaluation, otherwise we can't do a progressive rollout.

Reply:
this is for the admin API, so I wasn't sure how to approach this.  I can add it, but that means that as an admin API, we might have different functionality based on which customer is do the action for.

Reply:
yeah, this is alright, then, I guess. However, we have to be aware that we have to excplicitly enable a feature flag for the `global` user when we're not passing in a jira host URL, here. I just added the `global` user to our team segment so that these features are automatically enabled for us in production.